### DISCUSSION TITLE
setValue for useFieldArray doesn't work

### DISCUSSION THREAD

**Discussion by Moshappp** [react-hook-form/react-hook-form#9895] [Q&A] [answered]
```
> <Input
>   id={`educationList.${index}.major`}
>   placeholder="Major/Degree"
>   {...register(`educationList.${index}.major`, {
>     onChange: (e) => {
>       updateEducationList(
>         index,
>         "update",
>         "major",
>         e.target.value
>       );
>       setValue(
>         `educationList.${index}.major`,
>         e.target.value
>       );
>     },
>   })}
> />
> 
```

 `console.log(fields)`
 `setValue` doesn't update the `fields` array

**Comment by leapful** [comment, marked as answer]
@Moshappp 

Please read carefully the documentation here https://react-hook-form.com/api/usefieldarray

<img width="974" alt="Screenshot 2023-02-09 at 09 25 59" src="https://user-images.githubusercontent.com/18494222/217701646-ec4da307-3535-4a4c-820b-d3064e2eab7b.png">

Object `fields` only ensures default values, not current values. Please use `getValues` from `useForm` hook instead.
